### ISSUE TITLE
npm version and github releases do not match

### ISSUE THREAD

**Issue by odai-alali** [yariksav/vuetify-dialog#17]
Hi,
I have noticed two problems while using this library with npm and I think they are related:

### Versioning Problem
this package version on npm page https://www.npmjs.com/package/vuetify-dialog is 0.3.3

Why I am not able to find this release here on github?

### Disappears after update
`npm outdated` gives this output
```
Package                         Current  Wanted  Latest  Location
...
vuetify-dialog                   0.1.12  0.1.12   0.3.3  dis
...
```
and after `npm update`
```
Package                         Current  Wanted  Latest  Location
...
vuetify-dialog                  MISSING  0.1.12   0.3.3  dis
...
```
This behavior was repeated many times. The library is uninstalled after every `npm update` and I have to install it again.

**Comment by yariksav** [comment]
Hello, @odai-alali!  It's a weird situation. 
Releases tags on github and versions npm is not related to each other
Each has an own version marks. At all it possible to create npm package versions without any repository

I didn't put tags on github, because this is intermediate package realization.

I suppose something wrong with your package-lock.json file. Can you try to move  it to some dir (remove from project) and call `npm i` ? Maybe you have not installed package by npm, and putted mannualy to package.json?
Also you should use --save or --save-dev tags to tell npm how to remember package (I sure you know it)
And please use latest version on package, witch resolved some bugs `npm i vuetify-dialog@0.3.3`

Best Regards!

**Comment by odai-alali** [comment]
Thank you for the lightning fast answer @yariksav 

I do not have any experience with publishing npm, I just wanted to make sure that there is nothing wrong.

I updated to the latest version and called `npm update`. The problem seems to be solved after updating the package.

I'll reopen in case it happened again. 👍
